code/modules/mob/mob_helpers.dm

```diff
@@ -409,6 +409,26 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 		return
 	return TRUE
 
+/atom/proc/hasSiliconAccessInArea(mob/user)
+	return user && (issilicon(user) || (user.siliconaccesstoggle && (get_area(src) in user.siliconaccessareas)))
+
+/area/proc/hasSiliconAccessInArea(mob/user)
+	return user && (issilicon(user) || (user.siliconaccesstoggle && (src in user.siliconaccessareas)))
+
```

Comment:
Remove. `get_area(area)` will return the area itself anyway, on top of being a preprocessor macro, and having no proc overhead.

Reply:
already fixed before this